src/check-registry.js

```diff
@@ -71,6 +71,9 @@ async function checkUrlAvailability(url) {
         let templatesToRemove = 0;
         let templatesToUpdate = 0;
         for (const template of registry) {
+            if (template.status === TEMPLATE_STATUS_IN_VERIFICATION) {
```

Comment:
Are we doing a "checkUrlAvailability" check every time for all templates in the registry? For example, I am trying to add info about new template, Does it mean that all templates will be rechecked (I will have error mesage even if it's related to some another template)?